publicdb/default/management/commands/createfakedata.py

```diff
@@ -0,0 +1,78 @@
+from datetime import date
+
+import factory
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from tests.factories import histograms_factories, inforecords_factories, station_layout_factories
+
+
+class Command(BaseCommand):
+    help = 'Creates fake data for testing a local development server'
+
+    def handle(*args, **options):
+        if not settings.DEBUG:
+            raise Exception('Never run this on a production database!!')
+
+        with factory.Faker.override_default_locale('nl_NL'):
```

Comment:
The reason for this override is partly because we are based in the Netherlands, but also because the default locale does not have very many random city names. Those are used for the fake (sub)clusters and are required to be unique so this has a lower chance of conflicts.

In case you do get an error while running this command, just rerun it, it should not create even more clusters because it will try to find existing ones using the provided number.